### ISSUE TITLE
request: add jpegxl as a dependency

### ISSUE THREAD

**Issue by AndrewAnnex** [conda-forge/gdal-feedstock#752]
### Comment:

In contrast to #722, I'd like to request the addition of a new dependency of JPEG-XL (https://github.com/libjxl/libjxl). 

This would enable the jpegxl driver for GDAL and many neat compression options that brings to COGs.

**Comment by rouault** [comment]
cf https://github.com/conda-forge/imagecodecs-feedstock/issues/48 : the main issue would be to add libjxl itself in conda-forge

**Comment by AndrewAnnex** [comment]
I did see https://github.com/conda-forge/libjxl-split-feedstock, which seems to be building libjxl but haven't looked into it too much to see why the name isn't exactly libjxl.

**Comment by zklaus** [comment]
> perhaps because it issues separate libjxl and libjxl-tools packages?

Indeed, it is a multi-output package that creates those two packages, see

https://github.com/conda-forge/libjxl-split-feedstock/blob/e6b35d156d2213eda7635674b91a0e80dd9259eb/recipe/meta.yaml#L43

and

https://github.com/conda-forge/libjxl-split-feedstock/blob/e6b35d156d2213eda7635674b91a0e80dd9259eb/recipe/meta.yaml#L73
